src/test/suite/fixtures/recorded/actions/cloneUpVest2.yml

```diff
@@ -0,0 +1,31 @@
+languageId: plaintext
```

Comment:
This test looks identical to the other "clone up vest". What am I missing?

Reply:
It probably is. There is no way to redo this without making another recording

Reply:
So the plan is to delete this duplicate test then?

Reply:
Yes